scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -440,32 +505,225 @@ static void refresh_tune_params(void)
 	}
 }
 
+/*
+ * log2 helper functions taken from scx_lavd
+ */
+static inline __attribute__((always_inline)) u32 bpf_log2(u32 v)
```

Comment:
We should collect these into a math header later.

Reply:
Agreed, I'll do that in a follow on patch set just to avoid more churn on this one.